view/src/main/java/cmm/apps/esmorga/view/eventlist/mapper/EventListUiMapper.kt

```diff
@@ -2,14 +2,17 @@ package cmm.apps.esmorga.view.eventlist.mapper
 
 import cmm.apps.esmorga.domain.event.model.Event
 import cmm.apps.esmorga.view.eventlist.model.EventListUiModel
+import java.time.Instant
+import java.time.ZoneId
 import java.time.ZonedDateTime
 import java.time.format.DateTimeFormatter
 import java.util.TimeZone
 
 object EventListUiMapper {
 
-    fun formatDate(date: ZonedDateTime): String {
-        return date.format(DateTimeFormatter.ofPattern("d' de 'MMMM' a las 'HH:mm").withZone(TimeZone.getDefault().toZoneId()))
+    fun formatDate(date: Long): String {
+        val zonedDateTime: ZonedDateTime = ZonedDateTime.ofInstant(Instant.ofEpochMilli(date), ZoneId.of(TimeZone.getDefault().toZoneId().id))
```

Comment:
Can't we only use "TimeZone.getDefault().toZoneId()" instead "ZoneId.of(TimeZone.getDefault().toZoneId().id))"?